test/test_backend/db/test_utils.py

```diff
@@ -59,6 +60,17 @@ def test_select_failed_fetches(self):
         self.assertEqual(len(results), self.n1 + len(self.mock_data))
 
 
+class TestIdleConnections(unittest.TestCase):
+
+    def test_idle_connections(self, threshold=10):
```

Comment:
## Test
For checking acceptable number of idle connections within given time frame.

WIP